src/adapter/http/tests/harness/client_side_harness.rs

```diff
@@ -195,6 +196,10 @@ impl ClientSideHarness {
         b.add_value(kamu::utils::ipfs_wrapper::IpfsClient::default());
         b.add_value(IpfsGateway::default());
 
+        b.add::<DeviceCodeServiceImpl>();
+        b.add::<InMemoryDeviceCodeRepository>();
+        b.add::<DummyDatasetEntryIndexer>();
```

Comment:
Not sure I understood a need in dataset entry indexer here and in a few more test groups.
The authentication tasks should be totally independent on datasets domain.

Reply:
Deleted it -- it was being added since I mistakenly listed it as a dependency

Fixed